packages/plugins/ai/test/parse.test.ts

```
@@ -76,6 +115,9 @@ describe('AI Parsing Plugin', () => {
 		const result = await parseAI('The Friday after Thanksgiving', { anchor: anchorDate, timeZone: 'UTC', force: true });
 
 		expect(result).toBeInstanceOf(Tempo);
+		expect(result.constructor).toBe(Tempo);
+		expect(result.constructor === Tempo).toBe(true);
+		expect(result.format === result.format).toBe(true);
```

Biome (2.5.6)

[error] This comparison uses the same expression on both sides.

(lint/suspicious/noSelfCompare)